test/gtest/uct/uct_test.h

```diff
@@ -133,13 +133,13 @@ class uct_test : public testing::TestWithParam<const resource*>,
         struct sockaddr_user_data {
             sockaddr_user_data(uct_test *test, entity *entity,
                                unsigned ep_index) :
-                    m_test(test), m_entity(entity), m_ep_index(ep_index)
+                    test_p(test), entity_p(entity), entity_ep_index(ep_index)
             {
             }
 
-            uct_test *m_test;
-            entity   *m_entity;
-            unsigned m_ep_index;
+            uct_test *test_p;
+            entity   *entity_p;
```

Comment:
minor: i'd remove _p suffix for consistency (we use _p for pointer-to-handle func params in C code)

Reply:
there are 2 issues :)
1) need new names for args in ctor
2) compiler warns about `entity *entity` definition

maybe better to revert `m_*` prefixes?